CHANGES.md

```diff
@@ -1,5 +1,52 @@
+- `RepositoryDefinition.load_asset_value` and `AssetValueLoader.load_asset_value` now work with IO managers whose `load_input` implementation accesses the `op_def` and `name` attributes on the `InputContext`.
+- `RepositoryDefinition.load_asset_value` and `AssetValueLoader.load_asset_value` now respect the `DAGSTER_HOME` environment variable.
+- `InMemoryIOManager`, the `IOManager` that backs `mem_io_manager`, has been added to the public API.
+- The `multi_asset_sensor` (experimental) now supports marking individual partitioned materializations as “consumed”. Unconsumed materializations will appear in future calls to partitioned context methods.
+- The `build_multi_asset_sensor_context` testing method (experimental) now contains a flag to set the cursor to the newest events in the Dagster instance.
+- `TableSchema` now has a static constructor that enables building it from a dictionary of column names to column types.
+- Added a new CLI command `dagster run migrate-repository` which lets you migrate the run history for a given job from one repository to another.  This is useful to preserve run history for a job when you have renamed a repository, for example.
+- [dagit] The run timeline view now shows jobs grouped by repository, with each repository section collapsible. This feature was previously gated by a feature flag, and is now turned on for everyone.
+- [dagster-airbyte] Added option to specify custom request params to the Airbyte resource, which can be used for auth purposes.
+- [dagster-airbyte] When loading Airbyte assets from an instance or from YAML, a filter function can be specified to ignore certain connections.
+- [dagster-airflow] `DagsterCloudOperator` and `DagsterOperator` now support Airflow 2. Previously, installing the library on Airflow 2 would break due to an import error.
+- [dagster-duckdb] A new integration with DuckDB allows you to store op outputs and assets in an in-process database.
+
+### Bugfixes
+
+- [dagster-airbyte] Fixed generating assets for Airbyte normalization tables corresponding with nested union types.
+- [dagster-dbt] When running assets with `load_assets_from_...(..., use_build=True)`, AssetObservation events would be emitted for each test. These events would have metadata fields which shared names with the fields added to the AssetMaterialization events, causing confusing historical graphs for fields such as Compilation Time. This has been fixed.
+- [dagster-dbt] The name for the underlying op for `load_assets_from_...` was generated in a way which was non-deterministic for dbt projects which pulled in external packages, leading to errors when executing across multiple processes. This has been fixed.
+- Previously, if retries were exceeded when running with `execute_in_process`, no error would be raised. Now, a DagsterMaxRetriesExceededError will be launched off.
```

Comment:
Nitpick: I prefer to put the core changes above the library changes. Also, DagsterMaxRetriesExceededError could be monospaced.